docs/opensearch-sink-connector-config-options.rst

```diff
@@ -88,6 +88,13 @@ Data Conversion
   * Valid Values: ``insert``, ``upsert``
   * Importance: low
 
+``index.name.override``
```

Comment:
@porto88 the connector is not dealing with just one topic, the risk of misusing this feature (and unintended consequences of moving from one topic to many with this property being set) greatly outperform the benefits (in my opinion).

Reply:
this could be a mapping between topic and index name, similar to the Snowflake `snowflaketopic2table.map` that can be set to ` "snowflake.topic2table.map": "topic1:table1,topic2:table2"`

Reply:
The same thing can be achieved with the [RegexRouter](https://docs.confluent.io/platform/current/connect/transforms/regexrouter.html) smt while still supporting the multiple topics case @reta mentioned.